iwyu.cc

```diff
@@ -1601,12 +1601,10 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     // the namespace of the decl.  If we have 'using std::vector;' +
     // 'std::vector<int> foo;' we don't actually care about the
     // using-decl.
-    // TODO(csilvers): maybe just insert our own using declaration
-    // instead?  We can call it "Use what you use". :-)
```

Comment:
Ah yes, possibly this warrants more discussion, but I removed the comment because we can't actually do what it says. We could *possibly* remove the using from a source file, but not a header because in a header the using becomes an exported symbol.  In source, it might be possible, then IWYU would have different behaviour between source and header files, and I thought that would be undesirable.